components/Header.tsx

```diff
@@ -0,0 +1,18 @@
+import { Image } from "@chakra-ui/image";
```

Comment:
_(arbitrary comment location to start a thread)_

I got the same "Unhandled Runtime Error" that @wildmolasses mentioned [here](https://github.com/ScopeLift/railgun-shielder/pull/5#pullrequestreview-1234717650)

Reply:
Once I added the Alchemy API key this went away. Did you have the same experience @mds1?

Reply:
Ah yes that fixed it, marking this resolved